web/core/src/jsTest/kotlin/CSSStylesheetTests.kt

```diff
@@ -311,4 +309,22 @@ class CSSVariableTests {
             "Sibling selector isn't generated correctly"
         )
     }
+
+    @Test
+    fun testStringPlusCSSSelectorConcatenation() {
+        assertTrue("Concatenation of String + CSSSelector should be restricted.") {
```

Comment:
you can use assertFailsWith() in such cases if you like

Reply:
didn't know, thanks

Reply:
+1 for assertFailsWith